conf/jasminegraph-server.properties

```diff
@@ -25,20 +25,33 @@ org.jasminegraph.server.nworkers=2
 org.jasminegraph.server.npartitions=2
 #org.jasminegraph.server.runtime.location is the location where temporary files of JasmineGraph master is kept
 org.jasminegraph.server.runtime.location=./tmp
-org.jasminegraph.worker.path=/var/tmp/jasminegraph/
+org.jasminegraph.worker.path=/var/tmp/jasminegraph_fl/
 #Path to keep jasminegraph artifacts in order to copy them to remote locations. If this is not set then the artifacts in the
 #JASMINEGRAPH_HOME location will be copied instead.
 #if it is empty please leave a space, else it will cause an issue
 org.jasminegraph.artifact.path=
 #org.jasminegraph.partitioner.metis.bin is the location where the METIS graph partitioner's gpmetis executable is installed
-org.jasminegraph.partitioner.metis.bin=/home/ubuntu/software/metis-5.1.0/bin
+org.jasminegraph.partitioner.metis.bin=/home/damitha/ubuntu/software/metis/metis-5.1.0/bin
 #The following folder is the location where workers keep their data.
 #This is the location where the actual data storage takes place in JasmineGraph.
-org.jasminegraph.server.instance.datafolder=/var/tmp/jasminegraph-localstore
+org.jasminegraph.server.instance.datafolder=/var/tmp/jasminegraph-localstore_fl/
 #The folder path for keeping central stores for triangle count aggregation
-org.jasminegraph.server.instance.aggregatefolder=/var/tmp/jasminegraph-aggregate
-org.jasminegraph.server.instance.trainedmodelfolder=/var/tmp/jasminegraph-localstore/jasminegraph-local_trained_model_store
-org.jasminegraph.graphsage=/var/tmp/jasminegraph/GraphSAGE/graphsage/
+org.jasminegraph.server.instance.aggregatefolder=/var/tmp/jasminegraph-aggregate_fl
+org.jasminegraph.server.instance.trainedmodelfolder=/var/tmp/jasminegraph-localstore_fl/jasminegraph-local_trained_model_store/
+org.jasminegraph.graphsage=/var/tmp/jasminegraph_fl/GraphSAGE/graphsage/
+
+#--------------------------------------------------------------------------------
+#Federated
+#--------------------------------------------------------------------------------
+
+org.jasminegraph.federated.enabled=true
+org.jasminegraph.fl.location=/home/damitha/Documents/fed/federated-gcn/
```

Comment:
Lets correct these paths according to the docker version which we will develop

Reply:
Fixed in commit 8c750f4

Reply:
Done